appveyor.yml

```diff
@@ -20,36 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_NPY: 110
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      CONDA_NPY: 110
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_NPY: 111
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x64
-      CONDA_NPY: 111
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_NPY: 110
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_NPY: 110
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_NPY: 111
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_NPY: 111
-      CONDA_PY: 34
```

Comment:
Sorry just realized that I never answered this.

It shouldn't have anything to do with the tokens. Though `conda-smithy` was pretty sensitive to `.condarc`'s channel order. So that could have been an issue. We since fixed that in PR ( https://github.com/conda-forge/conda-smithy/pull/401 ).